olclient/cli.py

```diff
@@ -88,6 +88,16 @@ def main():
         config_tool.update(config)
         return "Successfully configured "
 
+    # prompt first time users to configure credentials for olclient
+    try:
+        if len(ia.config.get_config()) == 0:
```

Comment:
We may want to use `if not len(ia.config.get_config())` which is slightly more pythonic

Reply:
Much agreed, I shall commit the suggested change.
Thank you @mekarpeles